data-es2016plus.js

```diff
@@ -175,7 +175,6 @@ exports.tests = [
           es7shim:         true,
           typescript:      typescript.corejs,
           safari9:         true,
-          safari10:        true,
```

Comment:
`safari10` is `true` because of `safari9: true`. However, I don't know why it's a part of this PR instead of WebKit-related PRs.